filler.go

```diff
@@ -92,8 +91,17 @@ func (r *routeFiller) fill(ctx reflect.Value) error {
 	}
 	if r.matchedPathIndex != -1 {
 		i := r.routeParams[r.matchedPathIndex]
+		paths := r.reqURL[r.matchedPathIndex:]
+		path := bytes.NewBuffer([]byte(``))
+		for _, v := range paths {
+			path.WriteString(v)
+			path.WriteRune('/')
+		}
 		ctx.Elem().Field(i).
-			Set(reflect.ValueOf(r.matchedPath))
+			Set(reflect.ValueOf(
+				// remove last `/`
+				path.String()[:path.Len()-1],
```

Comment:
Can you help to try `String.Tirm` will affect the speed or not? 
If not, it will more readable and we can remove the comment.

Reply:
Not a lot, at least do not over 10%, so I think we could use it.